openstack/compute/v2/extensions/attachinterfaces/results.go

```diff
@@ -0,0 +1,43 @@
+package attachinterfaces
+
+import (
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// FixedIP represents a Fixed IP Address.
+type FixedIP struct {
+	SubnetID  string `json:"subnet_id"`
+	IPAddress string `json:"ip_address"`
+}
+
+// Interface represents a network interface on an instance.
+type Interface struct {
+	PortState string    `json:"port_state"`
+	FixedIPs  []FixedIP `json:"fixed_ips"`
+	PortID    string    `json:"port_id"`
+	NetID     string    `json:"net_id"`
+	MacAddr   string    `json:"mac_addr"`
```

Comment:
@FengyunPan I'm sorry, I just noticed this.

This should be `MACAddr`. There are several other areas in Gophercloud where MAC is used, so this will be consistent. It's better to change this now than later.